mypy/checker.py

```diff
@@ -131,6 +132,7 @@ class TypeChecker(NodeVisitor[None], CheckerPluginInterface):
     # Helper for type checking expressions
     expr_checker = None  # type: mypy.checkexpr.ExpressionChecker
 
+    tscope = None  # type: TargetScope
```

Comment:
I'd prefer renaming the `Scope` class in `checker.py` instead of importing `mypy.scope.Scope` as `TargetScope`. Having two classes with the same name (at least in their definition) can be pretty confusing.